charts/nri-metadata-injection/tests/securitycontext_test.yaml

```diff
@@ -0,0 +1,42 @@
+suite: test securityContext setup
```

Comment:
As this behavior is tested in the common library itself for [the pod](https://github.com/newrelic/helm-charts/blob/master/library/CHART-TEMPLATE/tests/common_images_security_context_pod_test.yaml) and [the container](https://github.com/newrelic/helm-charts/blob/master/library/CHART-TEMPLATE/tests/common_images_security_context_container_test.yaml) I would not test the security Context again.

I personally would test only if you are changing/overriding the behavior from the common library to test that the common library fits your needs for this particular use case but if you are using it as-is I would not care.

Reply:
Fair point! Redundant tests removed here: [a3d79f0](https://github.com/newrelic/k8s-metadata-injection/pull/163/commits/a3d79f01dfa8535d1c29c1535fe91eacbe949551)